### DISCUSSION TITLE
timescaledb support?

### DISCUSSION THREAD

**Discussion by liucougar** [teslamate-org/teslamate#1126] [Ideas]
This is a feature request, so not using the template.

Thanks for working on this great project. I just came across timescaledb, which is similar to influx but based on postgresql (11 and 12 for now).

as teslamate is using postgresql and most of the data are time series, I am wondering whether it makes sense to use timescaledb (it's an extension for postgresql) with teslamate?

timescaledb provides docker images: https://docs.timescale.com/v1.0/getting-started/installation/docker/installation-docker#postgis-docker

**Comment by wranglatang** [comment]
Im interested in using Influx DB instead of postgres. The main reason for me is just because I have multiple Influx DB's and no Postgres DB's so its more of a cleanliness / familiarity :)
